UPGRADE-4.0.md

```diff
@@ -151,6 +151,11 @@ HttpFoundation
     - `isInvalid`/`isSuccessful`/`isRedirection`/`isClientError`/`isServerError`
     - `isOk`/`isForbidden`/`isNotFound`/`isRedirect`/`isEmpty`
 
+ * The ability to check only for cacheable HTTP methods using `Request::isMethodSafe()` is
+   not supported anymore and throws an exception. Disable checking only for cacheable methods 
```

Comment:
I thought the argument would be uncommented as is so the exception would be thrown when omitting to pass the argument to `false` explicitly.